### PR TITLE
Berets

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Ported directly from oraclestation, with only a few minor additions.
unknown-109
I just now realize that my character has the worst hair possible for this, but it's too late to set that all up again and it's 2:30.

## Why It's Good For The Game
They look cool, and are more things to add to the beecoin store at some point. Could probably add them myself, but I'd rather they all be under one menu or the one specific to your job would be given instead of a bunch of separate entries.

## Changelog
:cl:
add: Berets to more departments
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Isn't there a var or something for hats to hide your hair while worn?

**Comment by ike709** [comment]
> Misspelling(s) of CENTCOM detected in code, please remove the extra M(s).

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -256,7 +270,94 @@
...
+/obj/item/clothing/head/beret/captain
+	name = "captain beret"
+	desc = "A lovely blue Captain beret with a gold and white insignia."
+	icon_state = "beret_captain"
+	armor = list(melee = 50, bullet = 30, laser = 20, energy = 10, bomb = 15, bio = 10, rad = 10, fire = 10, acid = 60)
+	strip_delay = 90
+
 #undef DRILL_DEFAULT
-#undef DRILL_SHOUTING
+#undef DRILL_SHOUTINIG
```
Comment (ike709) [inline code/modules/clothing/head/jobs.dm]: Bruh.

**Review by ike709 — CHANGES_REQUESTED** [review]
Address the things.

File: code/modules/clothing/head/jobs.dm (hunk 1 above)
Comment (MarkSuckerberg) [reply code/modules/clothing/head/jobs.dm]: it was at 3:30 in the morning cut me some slack

**Comment by MarkSuckerberg** [comment]
why does travis have a check for that I stg

**Comment by MarkSuckerberg** [comment]
what is an acclimator and why did it get added

**Comment by MarkSuckerberg** [comment]
betterberets
This is what they look like now. They just change your hairstyle to one that fits under, so it doesn't make you look like a cueball-head spaceman.

**Pull request merged**